internal/workflow/local_activities_test.go

```diff
@@ -287,3 +287,88 @@ func TestUpdateSIPLocalActivity(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdateBatchLocalActivity(t *testing.T) {
+	t.Parallel()
+
+	batchUUID := uuid.New()
+	startedAt := time.Date(2024, 6, 13, 17, 50, 13, 0, time.UTC)
+	completedAt := time.Date(2024, 6, 13, 17, 50, 14, 0, time.UTC)
+
+	for _, tt := range []struct {
+		name      string
+		params    *updateBatchLocalActivityParams
+		mockCalls func(context.Context, *ingest_fake.MockService)
+		wantErr   string
+	}{
+		{
+			name: "Updates a batch",
+			params: &updateBatchLocalActivityParams{
+				UUID:        batchUUID,
+				Status:      enums.BatchStatusProcessing,
+				StartedAt:   startedAt,
+				CompletedAt: completedAt,
+			},
+			mockCalls: func(ctx context.Context, svc *ingest_fake.MockService) {
+				svc.EXPECT().
+					UpdateBatch(
+						ctx,
+						batchUUID,
+						mockutil.Func(
+							"should update batch",
+							func(updater persistence.BatchUpdater) error {
+								b, err := updater(&datatypes.Batch{})
+								assert.NilError(t, err)
+								assert.DeepEqual(t, b.Status, enums.BatchStatusProcessing)
```

Comment:
Can't you just do:
```
assert.DeepEqual(t, b, &datatypes.Batch{
    Status:  enums.BatchStatusProcessing,
    StartedAt: startedAt,
    CompletedAt: completedAt
})
```

If not, then I think `assert.Equal(t, b.Status, enums.BatchStatusProcessing)` is sufficient.